Source/cmGlobalFastbuildGenerator.cxx

```diff
@@ -2202,10 +2219,10 @@ class cmGlobalFastbuildGenerator::Detail::Generation
 				{
 					context.fc.WriteArray("PreBuildDependencies",
 						Wrap(preBuildSteps, "'" + targetName + "-", "-" + configName + "'"),
-						"+");
+						"=");
```

Comment:
I think this should still be a "+" (and there should be a using statement above, see other comment)